client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -5,6 +5,29 @@ $design-button-primary-color: var( --color-primary );
 $get-started-button-primary-color: var( --studio-blue-50 );
 $white: var( --color-surface );
 
+
+.link-in-bio {
+	&.processing {
+		background: rgb( 162, 218, 254 );
+		position: fixed;
+		width: 100%;
+		height: 100%;
+
+		.step-container__content {
+		    display: flex;
+			justify-content: center;
```

Comment:
Based on the Figma and your screenshots, I think this is supposed to be aligned to the left with the flow title?

I addressed the flow title in this PR #66416 

Maybe we can merge that first and you can rebase to get the flow title in there?

Reply:
Sounds good to merge that one 1st and then rebase.
Thank you for letting me know about it! 👍